prometheus/src/test/scala/filodb/prometheus/parse/ParserSpec.scala

```diff
@@ -187,7 +187,7 @@ class ParserSpec extends FunSpec with Matchers {
     parseSuccessfully("count_values(\"value\",some_metric)")
     parseSuccessfully("sum without(and, by, avg, count, alert, annotations)(some_metric)")
     parseSuccessfully("sum_over_time(foo)")
-
+    parseSuccessfully("sum:some_metric:dataset:1m{_ws_=\"demo\", _ns_=\"test\"}")
```

Comment:
@TanviBhavsar can you please add a test case to expected logicalPlans? `Should be able to make logical plans for Series Expressions`

Reply:
Also lets add all the keywords to these test cases , for completeness. And I agree with Sher's suggestion too.

Reply:
Added test case to expected logicalPlan. Also added test case for other keywords